vpr/src/place/net_cost_handler.h

```diff
@@ -511,4 +512,17 @@ class NetCostHandler {
      */
     double get_net_wirelength_from_layer_bb_(ClusterNetId net_id);
 
+    template<typename BBT>
+    std::pair<double, double> get_chan_place_fac_(const BBT& bb) {
+        const int total_chanx_width = acc_chanx_width_[bb.ymax] - acc_chanx_width_[bb.ymin - 2];
```

Comment:
Could get rid of the -2 if we don't make the bounding box include channels on the left/below edges (just have to comment it).